.github/workflows/ci-cd.yml

```diff
@@ -153,6 +153,30 @@ jobs:
           python -m pip install `ls dist/*.tar.gz`
           pytest --dist loadscope --numprocesses auto tests/countries tests/financial
 
+  test-docs:
+    name: Test docs build
+    runs-on: ubuntu-latest
+    needs: test
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
+      - name: Set Up Python
+        uses: actions/setup-python@v5
+        with:
+          cache: pip
+          cache-dependency-path: requirements/docs.txt
+          python-version: '3.12'
+
+      - name: Install Dependencies
+        run: |
+          python -m pip install --requirement requirements/docs.txt
+          python -m pip install .
+
+      - name: Build docs
+        run: |
+          python -m sphinx -E -T -W -b html -D language=en -j auto -q docs/source docs/build
```

Comment:
It looks like we need to adjust `make doc` target and use it here instead.

Reply:
As I understand, this is actually the same command.

Reply:
Cool, so we can just use it as is then.